### PR TITLE
Update ManagementGroups SDK (2018-03-01-preview)

### PR DESCRIPTION
**Pull request by rajshah11**
## Description

Rest API Spec PRs 
https://github.com/Azure/azure-rest-api-specs/pull/3064
https://github.com/Azure/azure-rest-api-specs/pull/3115
---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/ManagementGroups/Management.ManagementGroups/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -10,8 +10,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Management Groups Library")]
 [assembly: AssemblyDescription("Provides functionality to handle management groups.")]
 
-[assembly: AssemblyVersion("1.1.0.0")]
-[assembly: AssemblyFileVersion("1.1.0.0")]
+[assembly: AssemblyVersion("1.2.0.0")]
+[assembly: AssemblyFileVersion("1.2.0.0")]
```
Comment (dsgouda) [inline src/SDKs/ManagementGroups/Management.ManagementGroups/Properties/AssemblyInfo.cs]: Only update the `AssemblyFileVersion` here

File: src/SDKs/ManagementGroups/Management.ManagementGroups/Management.ManagementGroups.csproj [hunk 2]
```diff
@@ -5,11 +5,11 @@
   
   <PropertyGroup>
 		<Description>Microsoft Azure Management Groups Library</Description>
-		<Version>1.1.0-preview</Version>
+		<Version>1.2.0-preview</Version>
 		<AssemblyName>Microsoft.Azure.Management.ManagementGroups</AssemblyName>
 		<PackageId>Microsoft.Azure.Management.ManagementGroups</PackageId>
 		<PackageTags>Microsoft Azure Management Groups;Management Groups;Groups;</PackageTags>
-		<PackageReleaseNotes>This is a public preview release of the Azure Management Groups SDK</PackageReleaseNotes>
+		<PackageReleaseNotes>This is a general availibility release of the Azure Management Groups SDK</PackageReleaseNotes>
```
Comment (dsgouda) [inline src/SDKs/ManagementGroups/Management.ManagementGroups/Management.ManagementGroups.csproj]: So, is this a stable release?

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please address comments

File: src/SDKs/ManagementGroups/Management.ManagementGroups/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (rajshah11) [reply src/SDKs/ManagementGroups/Management.ManagementGroups/Properties/AssemblyInfo.cs]: Okay. Thanks. Will make the change.

File: src/SDKs/ManagementGroups/Management.ManagementGroups/Management.ManagementGroups.csproj (hunk 2 above)
Comment (rajshah11) [reply src/SDKs/ManagementGroups/Management.ManagementGroups/Management.ManagementGroups.csproj]: Yes. We are GAing the end of this month.

Comment (dsgouda) [reply src/SDKs/ManagementGroups/Management.ManagementGroups/Management.ManagementGroups.csproj]: In that case, you will need to remove the `-preview` from the package version.

Comment (rajshah11) [reply src/SDKs/ManagementGroups/Management.ManagementGroups/Management.ManagementGroups.csproj]: Actually, if removing the preview tag is not a requirement for GAing a service we would like to keep it. Please let me know if we can keep the preview tag.

**Comment by dsgouda** [comment]
@rajshah11 if you received all the necessary information from @shahabhijeet I can approve this PR

**Review by dsgouda — APPROVED** [review]
LGTM subject to CIs passing

**Pull request merged**